### DISCUSSION TITLE
I would like to have a tool that can check D365 security roles against the MS restricted list of entities.

### DISCUSSION THREAD

**Discussion by AlexanderRuiter** [MscrmTools/XrmToolBox#1032] [Ideas]
Hello,

Introduction: 
The new Power App Per App license is considered to be the new cheap alternative; same as the Team Members license used to be.
Restriction however is that users may not be able to use one of the [restricted entities](https://docs.microsoft.com/en-us/powerapps/maker/data-platform/data-platform-restricted-entities).  
So it would be great to have a tool that checks users and/or roles for this.

**Comment by MscrmTools** [comment]
I'm not sure this is something doable easily...
There is no programmatic way to retrieve the list of restricted entities. So it means it would be necessary to minitor this list of restricted entities to update the tool itslelf... seems a bit complicated to me
